.github/workflows/benchmarks.yml

```diff
@@ -5,42 +5,29 @@ on:
   push: ~
 
 jobs:
-  unit-tests:
+  benchmarks:
     name: "benchmarks"
 
-    runs-on: ${{ matrix.operating-system }}
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        php-version:
+        php:
           - "8.1"
-        operating-system:
+        os:
           - "ubuntu-latest"
+          - "macos-latest"
```

Comment:
It seems that the extension is not loaded when running benchmarks on macOS, not sure why yet ( @veewee maybe you can have a look at that? )

Reply:
On mac, a compilation results in `libpsl.dylib` instead of `libpsl.so`.

The formats are interchangeable - so you should be able to add this step:

`mv libpsl.dylib libpsl.so || true`

Locally, This results in:

![Screenshot 2022-11-21 at 07 25 43](https://user-images.githubusercontent.com/1618158/202980005-e2f7b8e0-123f-4de9-a073-5c5f901bd1c3.png)

Reply:
Yeah, pcov was enabled..